.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.4.1
+script:
+  - bundle install
+  - bundle exec rake db:create --all
```

Comment:
Move all DB Setup in a `before_script:` block.
Also you don't need the `--all` of creating the DB and you can compact this in one line

`bundle exec rails db:create db:test:prepare` should do the trick in principle